<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_save">
         <property name="text">
          <string>保存关键点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_run">
         <property name="text">
          <string>运行</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupbox_tab">
      <property name="title">
       <string>手动调节:</string>
      </property>
      <layout class="QGridLayout" name="layout_tab">
       <item row="0" column="2">
        <widget class="QTabWidget" name="tab_widget_objs">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideLeft</enum>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="tabBarAutoHide">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="tab1">
          <attribute name="title">
           <string>物体1</string>
          </attribute>
          <layout class="QGridLayout" name="layout_subtab_1"/>
         </widget>
         <widget class="QWidget" name="tab2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupbox_outprint">
      <property name="title">
       <string>命令行:</string>
      </property>
      <layout class="QVBoxLayout" name="layou_text_edit">
       <item>
        <widget class="QPlainTextEdit" name="text_edit_outprint">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupbox_visualize_sets">
      <property name="title">
       <string>可视化选项:</string>
      </property>
      <layout class="QGridLayout" name="layout_visulize_sets">
       <item row="0" column="0">
        <widget class="QCheckBox" name="rbtn_points3d">
         <property name="text">
          <string>关键点</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="rbtn_backbone">
         <property name="text">
          <string>骨架</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="rbtn_points2d">
         <property name="text">
          <string>标记点</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="rbtn_axis">
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>坐标轴</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="rbtn_model">
         <property name="text">
          <string>模型</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
